scripts/shared/clusters.sh

```diff
@@ -113,6 +113,7 @@ function deploy_cni() {
 
 function deploy_weave_cni(){
     echo "Applying weave network..."
+    curl -sL "https://cloud.weave.works/k8s/net?k8s-version=v$k8s_version&env.IPALLOC_RANGE=${cluster_CIDRs[${cluster}]}" | sed 's!ghcr.io/weaveworks/launcher!weaveworks!' | kubectl apply -f -
```

Comment:
Shouldn’t L117 be removed?

Reply:
Doh, of course, thanks... My local version doesn’t have it (and works), I’m not sure what happened there.